packages/dd-trace/src/encode/0.4.js

```diff
@@ -13,6 +15,11 @@ float64Array[0] = -1
 
 const bigEndian = uInt8Float64Array[7] === 0
 
+const debugEncoding = isTrue(coalesce(
+  process.env.DD_TRACE_ENCODING_DEBUG,
+  false
+))
```

Comment:
This should probably go in the Config class and get passed into AgentEncoder at construction.

Reply:
👍 Should I create a new Config instance or pass an existing one? If the second, where do I get the existing instance from?

Reply:
Yes, something like that.

@rochdev What do you think is the best way to handle this?

Reply:
Actually this change breaks lots of tests that have assert on arguments passed to things. I don't think we want to make such a big change.

Reply:
I missed this comment but I replied as a review.